Vagrantfile

```diff
@@ -78,6 +78,7 @@ Vagrant.configure(2) do |config|
       grant all privileges on *.* to 'manoseimas'@'localhost';
       flush privileges;" || true
     sudo apt-get install -y nodejs nodejs-legacy npm
+    sudo apt-get install -y coffeescript
```

Comment:
Deployment script also needs to install coffeescript compiler. See https://github.com/ManoSeimas/manoseimas.lt/blob/master/deployment/deploy.yml#L41